src/client/resources/assets/snapper/lang/pt_br.json

```diff
@@ -0,0 +1,42 @@
+{
+  "key.snapper.panorama": "Faça Capturas de Tela panorâmicas",
+  "key.snapper.huge": "Faça Captura de Tela enorme",
+  "key.snapper.screenshot_menu": "Exibir Capturas de Tela",
+  "key.snapper.recent": "Abrir a última Captura de Tela",
+  "menu.snapper.screenshotmenu": "Capturas de Tela",
+  "menu.snapper.viewermenu": "Visualizador",
+  "menu.snapper.panorama": "Exibir Panorama",
+  "button.snapper.folder": "Abrir Pasta",
+  "button.snapper.panorama.tooltip": "Visualizar o panorama atual armazenado.",
+  "button.snapper.screenshots": "Exibir Capturas de Tela",
+  "button.snapper.delete": "Excluir",
+  "button.snapper.open": "Abrir Imagem",
+  "button.snapper.view": "Exibir Imagem",
+  "button.snapper.copy": "Copiar",
+  "button.snapper.rename": "Renomear",
+  "text.snapper.created": "Criado",
+  "text.snapper.generic": "Captura de Tela",
+  "text.snapper.loading": "Carregando Capturas de Tela",
+  "text.snapper.empty": "Faça uma Captura de Tela no jogo para vê-la aqui!",
+  "text.snapper.panorama_encourage": "Faça um panorama no jogo para vê-lo aqui!",
+  "text.snapper.panorama_success": "Panorama salvo. Visualize pressionando %s",
+  "text.snapper.panorama_failure_iris": "Devido a um problema no Iris, os panoramas não podem ser tirados enquanto ele estiver instalado.",
```

Comment:
"Tirar" (tirados) is an informal term which means "Take [pictures]" in this context. It would be better to use "Capturar" (capturados) instead.